Tests/SwiftDocCTests/Infrastructure/DocumentationContext/DocumentationContextTests.swift

```diff
@@ -4258,101 +4258,121 @@ let expected = """
-        let initializerReferences = try initializerIdentifiers.map { try XCTUnwrap(context.symbolIndex[$0]) }
-        let staticMethodsReferences = try staticMethodIdentifiers.map { try XCTUnwrap(context.symbolIndex[$0]) }
-        let operatorReferences = try operatorIdentifiers.map { try XCTUnwrap(context.symbolIndex[$0]) }
-        let subscriptReferences = try subscriptIdentifiers.map { try XCTUnwrap(context.symbolIndex[$0]) }
-        let functionReferences = try functionIdentifiers.map { try XCTUnwrap(context.symbolIndex[$0]) }
-        let macroReferences = try macroIdentifiers.map { try XCTUnwrap(context.symbolIndex[$0]) }
-        
-        try checkContainsSiblingOverloads(for: methodsReferences, using: context)
-        try checkContainsSiblingOverloads(for: initializerReferences, using: context)
-        try checkContainsSiblingOverloads(for: staticMethodsReferences, using: context)
-        try checkContainsSiblingOverloads(for: operatorReferences, using: context)
-        try checkContainsSiblingOverloads(for: subscriptReferences, using: context)
-        try checkContainsSiblingOverloads(for: functionReferences, using: context)
-        try checkContainsSiblingOverloads(for: macroReferences, using: context)
     }
     
     // We do not want to add overload behavior for some symbol kinds, even if they are collisions in the link resolver.
     func testContextDoesNotRecognizeUnoverloadableSymbolKinds() throws {
         enableFeatureFlag(\.isExperimentalOverloadedSymbolPresentationEnabled)
 
-        let (_, context) = try testBundleAndContext(named: "OverloadedSymbols")
-
-        let structIdentifiers = ["s:8ShapeKit22overloadedparentstructV",
-                                 "s:8ShapeKit22OverloadedParentStructV"]
+        enableFeatureFlag(\.isExperimentalOverloadedSymbolPresentationEnabled)
         
-        let structReferences = try structIdentifiers.map { try XCTUnwrap(context.symbolIndex[$0]) }
+        func makeSymbol(identifier: String, kind: SymbolGraph.Symbol.KindIdentifier) -> SymbolGraph.Symbol {
+            return SymbolGraph.Symbol(
+                identifier: .init(precise: identifier, interfaceLanguage: SourceLanguage.swift.id),
+                names: .init(title: "SymbolName", navigator: nil, subHeading: nil, prose: nil),
+                pathComponents: ["SymbolName"],
+                docComment: nil,
+                accessLevel: .public,
+                kind: .init(parsedIdentifier: kind, displayName: "Kind Display Name"),
+                mixins: [:]
+            )
+        }
         
-        for reference in structReferences {
-            let documentationNode = try XCTUnwrap(context.documentationCache[reference])
-            let overloadedSymbol = try XCTUnwrap(documentationNode.semantic as? Symbol)
-            XCTAssertNil(overloadedSymbol.overloadsVariants.firstValue)
+        let overloadableKindIDs = SymbolGraph.Symbol.KindIdentifier.allCases.filter { !$0.isOverloadableKind }
```

Comment:
This would be easier to understand if the variable were named something like `nonOverloadableKindIDs`, since this test is checking symbol kinds that (currently) don't have overloads.

Reply:
Fixed in https://github.com/apple/swift-docc/pull/819/commits/6ebbd7392f834b27545893dbc1946c17a9a8c8ab